nix/deps/nodejs-patched/default.nix

```diff
@@ -4,16 +4,17 @@
 
 { stdenv, deps, nodejs, patchMavenSources }:
 
+let
+  patchesDir = ./../../../patches;
```

Comment:
wondering if should I move `patches` directory inside `nodejs-patched` or keep it in root 🤔
In most projects patches remain in root

Reply:
maybe `$GITROOT` could be used to make this neater?

Reply:
No, Nix paths have to be relative here. I don't mind them being at repo root, but they would also make sense here.

Reply:
by here do you mean instead of `status-mobile/patches` we could store patches in `status-mobile/nix/deps/nodejs-patched/patches` ?
I don't mind actually, I'd be more than happy to get rid of the "./../../../" from the `patchesDir` variable :D